### PR TITLE
Rans abl2

### PR DESCRIPTION
**Pull request by janiadcock**
**Pull-request type:**
- [ ] Bug fix 
- [x] Documentation update
- [ ] Feature enhancement

## Description of the pull-request

Correct reference formatting in documentation and test case comments.

## Checklist

*All pull requests*
- [x] Builds successfully (*must test on at least one system/compiler combination*)
  - Operating systems 
    - [x] Linux
    - [ ] MacOS
  - Compilers 
    - [ ] GCC
    - [ ] LLVM/Clang
    - [x] Intel compilers
    - [ ] NVIDIA CUDA
- [x] Compiles without warnings
- [x] Passes all unit tests
- [ ] Passes all regression tests
- [ ] Documentation builds without errors

*For new code updates*
- [ ] Documentation updates - additions to user, theory, and verification manuals
- [ ] New unit tests providing coverage for new code, bug fixes etc.
- [ ] New regression tests exercising the new code

### CODE REVIEW

File: docs/references/references.bib [hunk 1]
```diff
@@ -355,15 +355,17 @@ @article{Haering-etal:2020
   url       = {http://arxiv.org/abs/2006.13118},
 }
 
-@article{Bautista2015,
+<<<<<<< HEAD
+@article{Bautista:2015,
```
Comment (sayerhs) [inline docs/references/references.bib:359]:
```suggestion
@article{Bautista:2015,
```

File: docs/references/references.bib [hunk 2]
```diff
@@ -355,15 +355,17 @@ @article{Haering-etal:2020
...
   author = {Mary C. Bautista and Louis Dufresne and Christian Masson},
   title = {Hybrid turbulence models for atmospheric flow; A proper comparison with RANS models},
   journal = {E3S Web of Conferences},
   year = {2015},
   url = {https://www.e3s-conferences.org/articles/e3sconf/abs/2015/01/e3sconf_sowe2014_03001/e3sconf_sowe2014_03001.html},
 }
 
-@phdthesis{Bautista2011,
+<<<<<<< HEAD
+@phdthesis{Bautista:2011,
```
Comment (sayerhs) [inline docs/references/references.bib:368]:
```suggestion
@phdthesis{Bautista:2011,
```

File: docs/source/theory/boundaryConditions.rst [hunk 3]
```diff
@@ -802,17 +802,17 @@ SST RANS Model for Atmospheric Boundary Layer
 
 The following boundary conditions simulate the Atmospheric Boundary Layer, as described in Bautista, :cite:`Bautista:2011` and :cite:`Bautista:2015`. The Nalu-Wind SST RANS implementation matches the Monin-Obukhov profile when used with the model constants from Table-A I-1 (Boundreault, 2011) in :cite:`Bautista:2011` and the meshing method described  in :cite:`Bautista:2015`. The mesh described in :cite:`Bautista:2015` gives the Monin-Obukhov profile for roughness height 0.1. When the roughness height is decreased, the mesh must be refined near the wall. For example, for the :cite:`Bautista:2015` ABL test case using roughness height 0.001 instead of 0.1, the mesh size needs to be halved near the wall. 
     
-The :math:`k` and :math:`\omega` boundary conditions are the same as in the :ref:`k-omega SST boundary conditions` <_kwsst_high_re_bc>:
+The :math:`k` and :math:`\omega` boundary conditions are the same as in the :ref:`k-omega SST boundary conditions <_kwsst_high_re_bc>`:
```
Comment (sayerhs) [inline docs/source/theory/boundaryConditions.rst:805]:
```suggestion
The :math:`k` and :math:`\omega` boundary conditions are the same as in the :ref:`k-omega SST boundary conditions <kwsst_high_re_bc>`:
```

File: docs/source/theory/boundaryConditions.rst [hunk 4]
```diff
@@ -802,17 +802,17 @@ SST RANS Model for Atmospheric Boundary Layer
...
-.. math:: k = \frac{u_{\tau}^{2}}{\sqrt{\beta^*}}.
+.. math:: k = \frac{u_{\tau}^{2}}{\sqrt{\beta^*}},
 
 and 
 
-.. math:: \omega = \frac{u_{\tau}}{\sqrt{\beta^*} \kappa y},
+.. math:: \omega = \frac{u_{\tau}}{\sqrt{\beta^*} \kappa y}.
 
-The momentum boundary condition is a no-slip Dirichlet condition, :math:`u_i=0`, as described in the :ref:`momentum wall boundary conditions` <_wall_bc_momentum>.
+The momentum boundary condition is a no-slip Dirichlet condition, :math:`u_i=0`, as described in the :ref:`momentum wall boundary conditions <_wall_bc_momentum>`.
```
Comment (sayerhs) [inline docs/source/theory/boundaryConditions.rst:813]:
```suggestion
The momentum boundary condition is a no-slip Dirichlet condition, :math:`u_i=0`, as described in the :ref:`momentum wall boundary conditions <wall_bc_momentum>`.
```

File: docs/source/theory/boundaryConditions.rst [hunk 5]
```diff
@@ -802,17 +802,17 @@ SST RANS Model for Atmospheric Boundary Layer
...
 
-The streamwise and spanwise boundary conditions are periodic, as described in the :ref:`periodic boundary conditions` <_periodic_bc>.
+The streamwise and spanwise boundary conditions are periodic, as described in the :ref:`periodic boundary conditions <_periodic_bc>`.
```
Comment (sayerhs) [inline docs/source/theory/boundaryConditions.rst:815]:
```suggestion
The streamwise and spanwise boundary conditions are periodic, as described in the :ref:`periodic boundary conditions <periodic_bc>`.
```

**Review by sayerhs — CHANGES_REQUESTED** [review]

**Review by sayerhs — APPROVED** [review]
